configs/default/NEUT-FLOWBOX.config

```diff
@@ -0,0 +1,64 @@
+# Betaflight / STM32F411 (S411) 4.3.0 May 20 2021 / 03:37:19 (e0ad71db2) MSP API: 1.44
+
+board_name FLOWBOX
+manufacturer_id NEUT
+
+# resources
+resource BEEPER 1 B01
+resource LED_STRIP 1 A15
+resource SERIAL_TX 1 A09
+resource SERIAL_TX 2 A02
+resource SERIAL_RX 1 A10
+resource SERIAL_RX 2 A03
+resource LED 1 C13
+resource SPI_SCK 1 A05
+resource SPI_SCK 3 B03
+resource SPI_MISO 1 A06
+resource SPI_MISO 3 B04
+resource SPI_MOSI 1 A07
+resource SPI_MOSI 3 B05
+resource FLASH_CS 1 A15
+resource GYRO_EXTI 1 A01
+resource GYRO_CS 1 A04
+resource PWM 1 B10
+
+# timer
+timer B01 AF2
+# pin B01: TIM3 CH4 (AF2)
+timer A15 AF1
```

Comment:
What's the timers / DMAs used for in addition to the beeper?
Also, the PWM pin will need a timer to work.

Reply:
For serial receivers. Thanks for your hint, will update.